com.unity.uiwidgets/Runtime/rendering/texture.cs

```diff
@@ -43,11 +53,39 @@ public class TextureBox : RenderBox {
             return true;
         }
 
+        void _scheduleAppFrame() {
+            if (_frameCallbackScheduled || textureId == null) {
+                return;
+            }
+            
+            _frameCallbackScheduled = true;
+            SchedulerBinding.instance.scheduleFrameCallback(_handleAppFrame);
+        }
+
+        void _handleAppFrame(TimeSpan timestamp) {
+            _frameCallbackScheduled = false;
+            if (textureId == null) {
+                return;
+            }
+
+            TimeSpan delay = TimeSpan.Zero;
+            delay = new TimeSpan((long) (delay.Ticks * scheduler_.timeDilation));
+            _timer = Timer.create(delay, () => _scheduleAppFrame());
+        }
+
         public override void paint(PaintingContext context, Offset offset) {
             if (_textureId == null) {
+                if (_texture != null) {
```

Comment:
personal habits, I prefer `if(_textureId == null && _texture != null)`,

Reply:
good point. I will fix it!